lib/function_loader.js

```diff
@@ -32,6 +32,7 @@ function checkConflicts(obj1, obj2) {
 module.exports = function(eyeglass, sass, options, existingFunctions) {
   var root = options.root;
   var functions = {};
+  hash.merge(functions, require("./functions")(eyeglass, sass));
```

Comment:
Huh, I actually had the opposite inclination. Since this isn't a plugin, being able to override it seemed like a good usecase.